### DISCUSSION TITLE
[Bug] Unable to switch from yarn 2 to yarn 1

### DISCUSSION THREAD

**Discussion by ghost** [yarnpkg/berry#2971] [Q&A] [answered]
I've uninstalled yarn 2 via `sudo npm uninstall -g yarn`. Running `yarn -v` returns `bash: /usr/bin/yarn: No such file or directory`.

So far, so good. Then I ran `sudo npm install -g 'yarn@1.22.5'`:
```
$ sudo npm install -g 'yarn@1.22.5'
/usr/bin/yarnpkg -> /usr/lib/node_modules/yarn/bin/yarn.js
/usr/bin/yarn -> /usr/lib/node_modules/yarn/bin/yarn.js
+ yarn@1.22.5
```
 but whenever I run `yarn -v` it gives me `2.4.1`.

What am I doing wrong?

**Comment by merceyz** [comment]
You need to revert the actions done by `yarn set version ...`
https://yarnpkg.com/getting-started/install#per-project-install

**Reply by ghost** [reply]
Unfortunately this didn't work.
```
$ yarn set version 1.22.5
➤ YN0000: Downloading https://github.com/yarnpkg/yarn/releases/download/v1.22.5/yarn-1.22.5.js
➤ YN0000: Saving the new release in .yarn/releases/yarn-1.22.5.cjs
➤ YN0000: Done in 1s 436ms
```
But `yarn -v` still returns `2.4.1`, even though I've closed and reopened the terminal.

**Comment by ghost** [comment, marked as answer]
Solved. Besides running `sudo npm uninstall -g yarn`, I've also ran:
`sudo apt remove yarn && sudo apt purge yarn`
and
`sudo rm -rf /usr/local/bin/yarn && sudo rm -rf /usr/local/bin/yarn.lock`
Then I reinstalled it with `sudo npm install -g 'yarn@1.22.5'` and ran `yarn set version 1.22.5` in my project folder.
